frontend/src/app/libs/common-components/components/event-tile/event-tile.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
+import { HintMenuItem } from '../../models/hint-menu-item';
+
+@Component({
+  selector: 'app-event-tile',
+  templateUrl: './event-tile.component.html',
+  styleUrls: ['./event-tile.component.scss']
+})
+export class EventTileComponent {
+
+	@Input()
+	public isDraft: boolean;
+
+	@Input()
+	public name: string;
+
+	@Input()
```

Comment:
StartDate and EndDate is enough.
Please get start time and end time from this variables.

Reply:
Please remove start/end Time params.